README.textile

```diff
@@ -69,6 +69,10 @@ the problem. Also see "this FAQ":https://web.archive.org/web/20140512193354/http
 
 On OS X with Homebrew, try [re]installing pkg-config:
 
+On Linux(you are awesome), try
+
+bc. sudo apt-get install libmagickwand-dev
+
```

Comment:
All of these changes have been put between the introductory line about OS X and the actual instructions for OS X, so your additions are squeezed in between. Either put it before the OS X instructions or after.